work-item-link-category-blackbox_test.go

```diff
@@ -346,6 +345,11 @@ func TestSuiteWorkItemLinkCategory(t *testing.T) {
 }
 
 func getWorkItemLinkCategoryTestData(t *testing.T) []testSecureAPI {
+	configuration, err := config.NewConfigurationData("config.yaml")
```

Comment:
I'm getting a bit nervous when I see this `"config.yaml"` string in so many locations. Can we make this a function in the `configuration` package that returns a default path depending on from where the package is called?